_posts/2019-01-23-lecture-03.md

```diff
@@ -0,0 +1,471 @@
+        </div>
+    </div>
+</figure>
+<figure id="clique-example" class="l-body">
+    <div class="row">
+        <div class="col two">
+            <img src="{{ 'assets/img/notes/lecture-03/clique_example_max_clique.PNG' | relative_url }}" />
+        </div>
+    </div>
+</figure>
+<figure id="clique-example" class="l-body">
+    <div class="row">
+        <div class="col two">
+            <img src="{{ 'assets/img/notes/lecture-03/clique_example_sub_clique.PNG' | relative_url }}" />
+        </div>
+    </div>
+</figure>
+
+### Potential Functions
+Each clique can be associated with a **potential function** $$\psi$$, which can be understood as a provisional function of its arguments that assigns a pre-probabilistic score of their joint distribution. This potential function can be somewhat arbitrary, but must be non-negative.
+
+Why cliques? Each component of the clique contributes to the overall potential function.
+#### Example
+<figure id="potential-example" class="l-body">
+    <div class="row">
+        <div class="col one">
+            <img src="{{ 'assets/img/notes/lecture-03/potential_func_example.PNG' | relative_url }}" />
+        </div>
+    </div>
+    <figcaption>
+    <strong>An example used to illustrate potential functions. Both X1 and X2 are binary.</strong>
+    </figcaption>
+</figure>
+For $$\psi_c(X_1, X_2)$$,
+<figure id="potential-example" class="l-body">
+    <div class="row">
+        <div class="col one">
+            <img src="{{ 'assets/img/notes/lecture-03/potential_func_table.PNG' | relative_url }}" />
+        </div>
+    </div>
+    <figcaption>
+    <strong>This table represents the potential function values given combinations of random variable inputs.</strong>
+    </figcaption>
+</figure>
+
+
+Potential functions are not necessarily probabilistic:
+<figure id="potential-example" class="l-body">
+    <div class="row">
+        <div class="col one">
+            <img src="{{ 'assets/img/notes/lecture-03/potential_func_illustration.PNG' | relative_url }}" />
+        </div>
+    </div>
+    <figcaption>
+    <strong>Another example used to illustrate potential functions.</strong>
+    </figcaption>
+</figure>
+This model implies that $$X \perp Z | Y$$. This independence statement implies (by definition) that the joint must factorize as:
+<d-math block>
+\begin{aligned}
+    p(x,y,z)&=p(y)p(x|y)p(z|y) \\
+    &=p(x,y)p(z|y) \\
+    &=p(x|y)p(y,z)
+\end{aligned}
+</d-math>
+Probability distributions can be used as potential functions. However, in this case, we cannot let all potentials be either marginal probabilities or conditional probabilities. So the potential function for this graph cannot be probability distributions.
+
+### Gibbs Distribution and Undirected Graphical Model Definition
+Definition: Given an undirected graph $$H$$ and clique potentials functions $$\psi_C$$ associated with cliques of $$H$$, we say $$P(X_1, ..., X_n)$$ is a **Gibbs distribution** over $$H$$ if it can be represented as
```

Comment:
```suggestion
Given an undirected graph $$H$$ and clique potentials functions $$\psi_C$$ associated with cliques of $$H$$, we say $$P(X_1, ..., X_n)$$ is a **Gibbs distribution** over $$H$$ if it can be represented as
```